src/sync/SplitUpdateWorker/index.js

```diff
@@ -23,26 +25,29 @@ export default class SplitUpdateWorker {
     // requests are failing due to some network or server issue.
     if (this.maxChangeNumber > this.splitStorage.getChangeNumber() && this.attempts <= 1) {
       this.attempts++;
-      this.splitProducer.synchronizeSplits().then(() => {
+      this.splitProducer.synchronizeSplits(this.isSplitKill).then(() => {
         this.__handleSplitUpdateCall();
       });
     } else {
       this.maxChangeNumber = 0;
+      this.isSplitKill = false;
```

Comment:
Does it need to be an instance property?

Reply:
it was removed